runtime/cere/src/lib.rs

```diff
@@ -1448,7 +1472,7 @@ pub mod migrations {
 		pallet_ddc_customers::migration::v2::MigrateToV2<Runtime>,
 		pallet_ddc_clusters::migrations::v3::MigrateToV3<Runtime>,
 		pallet_ddc_nodes::migrations::v1::MigrateToV1<Runtime>,
-		UpgradeSessionKeys,
+		//UpgradeSessionKeys,
```

Comment:
There is no reason to comment out `UpgradeSessionKeys` as the whole `migrations::Unreleased` just should not be included to migrations for QANTE as I sai above